examples/lighting-app/linux/include/Options.h

```diff
@@ -0,0 +1,48 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      Support functions for parsing command-line arguments.
+ *
+ */
+
+#pragma once
+
+#include <cstdint>
+
+#include <core/CHIPError.h>
+
+class LinuxDeviceOptions
+{
+public:
+    uint32_t mBleDevice;
```

Comment:
if we set:

```
  public:
     uint32_t mBleDevice = 0;
```

we may not need to declare a constructor.

Would a class with all public data members be better modelled as a struct?

Reply:
That's true, fixed in the latest commit.